go/multiorch/coordinator/coordinator.go

```diff
@@ -94,7 +94,14 @@ func (c *Coordinator) AppointLeader(ctx context.Context, shardID string, cohort
 
 	// Stage 6: Propagate (setup replication within shard)
 	c.logger.InfoContext(ctx, "Stage 6: Propagating replication", "shard", shardID)
-	if err := c.Propagate(ctx, candidate, standbys, term, quorumRule); err != nil {
+
+	// Reconstruct the recruited list (nodes that accepted the term)
```

Comment:
under which scenarios would the cohort be different than recruited? Maybe good to add a comment around that.